SU-HSDO/suhumsci
# Hsd8 667 hires image effect

# READY FOR REVIEW

# Summary
- Created and configured a hires image effect similar to D7

# Needed By (Date)
- 7/17

# Urgency
- low

# Steps to Test
1. Checkout this branch
1. sync any site or install a fresh site
1. create a basic page with a text area
1. add an bigger image via media button in the wysiwyg.
1. Choose one of the image styles "Large", "Medium" or "Small"
1. save the page
1. Review the displayed image. The attributes on the img tag should be 1/2 as big as the image is naturally. I use firefox to get the original image size when i paste the image path into the browser and right click and choose "View image info". or `identify http://domain.loc/path-to-image.jpg`

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
